runtime/help/options.md

```diff
@@ -201,11 +195,11 @@ Here are the available options:
 
     default value: `false`
 
-* `incsearch`: enable incremental search in "Find" prompt (matching as you type).
+* `ignorecase`: perform case-insensitive searches.
 
     default value: `true`
 
-* `ignorecase`: perform case-insensitive searches.
+* `incsearch`: enable incremental search in "Find" prompt (matching as you type).
```

Comment:
Also valid for:
https://github.com/zyedidia/micro/blob/dc186429852572fd9ce419c02822d7ca61629cb8/internal/config/settings.go#L72-L73